### PR TITLE
Create epso.md

### PR DESCRIPTION
**Pull request by phwegner**

### CODE REVIEW

**Comment by matentzn** [comment]
@phwegner 

Thanks, you are missing a few required metadata elements.

for example contact. Example:

```
contact:
  email: entiminae@gmail.com
  label: Jennifer C. Girón
```

Check this as an example to figure out what you are missing:
https://github.com/phwegner/OBOFoundry.github.io/edit/master/ontology/aism.md

**Comment by matentzn** [comment]
Also, your citation is not really "real" - is there anything like a proper DOI link or paper that can be cited?

**Comment by cthoyt** [comment]
@matentzn are there recommendations for citations? like if there's no paper associated with an ontology, would a zenodo DOI be preferred?

**Comment by matentzn** [comment]
We dont really have any requirements to that end, but I would think the same principles apply as in regular scientific publishing. A DOI is fine, but I guess the output of [GitHub Citation Files](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files) is also fine.

**Comment by cthoyt** [comment]
Then it's basically the same as just putting the repo, but formatted. Maybe we should suggest people are using zenodo for long term storage, either way.

Three other things:

1. This should be linked to #1514
2. Speaking of repository citations, this repo seems to be in the personal account of @phwegner. It hasn't yet written who the responsible person is. @phwegner are you individually responsible for this in a long term capacity? Are you a student will soon move on and your organization will maintain it after?
3. Further problematic statement:

    > The Epilepsy Ontology will be maintained by Fraunhofer SCAI for at least 3 years from now on.

   Does this mean after 3 years and one day it will be abandoned? How does OBO Foundry handle promises like this?

**Comment by phwegner** [comment]
@matentzn  We don't have any publication yet but one will be published in the next weeks. 

@cthoyt  I changed the contact to Alpha Tom Kodamullil's name and email. The repo is in fact my personal github but this will be maintained by me and the rest of the team.

**Comment by phwegner** [comment]
@matentzn  @cthoyt GitHUb says there are reviews required. Is there anything I have to do to trigger that ? Are there any other updates on the submission process ?

**Review by matentzn — APPROVED** [review]

File: ontology/epso.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+tracker: https://github.com/phwegner/Epilepsyontology/issues
+dependencies:
+  - id: BFO
+products:
+  - id: epso.owl
+  - id: EPSO_merged.owl
+contact:
+  email: alpha.tom.kodamullil@scai.fraunhofer.de
+  label: Alpha Tom Kodamullil
+
```
Comment (cthoyt) [inline ontology/epso.md:17]: A GitHub handle for the contact is now requested in [FP-011](http://www.obofoundry.org/principles/fp-011-locus-of-authority.html) using the `github` key

Comment (cthoyt) [inline ontology/epso.md:17]: please add a `repository` key

**Review by cthoyt — CHANGES_REQUESTED** [review]
Added 2 comments on things that need to be changed in the metadata

**Comment by phwegner** [comment]
@cthoyt I added the repository ssh key but I'm not sure what you mean by github key

File: ontology/epso.md (hunk 1 above)
Comment (cthoyt) [reply ontology/epso.md:19]: more specifically:
```yaml
contact:
  email: alpha.tom.kodamullil@scai.fraunhofer.de
  label: Alpha Tom Kodamullil
  github: <handle goes here>
```

**Comment by phwegner** [comment]
@cthoyt  okay I changed it

File: ontology/epso.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  - id: BFO
+products:
+  - id: epso.owl
+  - id: EPSO_merged.owl
+contact:
+  email: alpha.tom.kodamullil@scai.fraunhofer.de
+  label: Alpha Tom Kodamullil
+  github: akodamullil
+  
+repository: git@github.com:phwegner/Epilepsyontology.git
```
Comment (matentzn) [inline ontology/epso.md:19]: should be https://github.com/phwegner/Epilepsyontology

Comment (phwegner) [reply ontology/epso.md]: @matentzn changed it

**Comment by matentzn** [comment]
thanks @phwegner

**Comment by phwegner** [comment]
@matentzn I see that everything is checked. Does that mean we get an URL for the Ontology soon ?
And thank you for you help @matentzn @cthoyt

**Comment by cthoyt** [comment]
No license declaration?

**Comment by matentzn** [comment]
How did that pass through QC? no license declaration?

**Comment by matentzn** [comment]
@phwegner you already "have" your purl, but if you want the redirects to work, you need to finish this: https://github.com/OBOFoundry/purl.obolibrary.org/pull/798

Just let me know when you are done.

**Comment by phwegner** [comment]
@cthoyt I just added a license

**Pull request merged**